src/main/java/com/suse/saltstack/netapi/client/SaltStackClient.java

```diff
@@ -385,4 +386,19 @@ public Job call() throws SaltStackException {
         };
         return executor.submit(callable);
     }
+
+  public Stats stats() throws SaltStackException {
```

Comment:
I am wondering if there is needed synchronous `stats()` at all? Maybe `statsAsync()` can be `stats()` and `stats()` can be removed completely? What is your opinion here?